src/screens/Lending/WithdrawTransactionConfirm.js

```diff
@@ -80,10 +76,10 @@ const WithdrawTransactionConfirm = ({
       accountAddress,
       withdrawAmount,
       depositedAsset,
-      txFeeInfo?.fee || new BigNumber(0),
+      feeInfo?.fee || new BigNumber(0),
```

Comment:
Same question here as before - as this might repeat, i won't point it out again.

Reply:
Removed per answer above 👍